ws/static/css/layout.css

```diff
@@ -152,6 +152,10 @@ admin-trip-signups .signup-list .signup-summary .label {
   margin-right: 5px;
 }
 
+admin-trip-signups .signup-list-feedback {
+  white-space: pre-wrap;
```

Comment:
This probably isn't the property you want. It honors whitespace _exactly_, so you can get weird stuff like:

### `pre-wrap`
<img width="376" alt="image" src="https://user-images.githubusercontent.com/901169/51158871-4e75f080-183b-11e9-82b9-42137e527465.png">

### `pre-line`
Using `pre-line` is closer, but still has the problem of not condensing newlines, which I've discussed elsewhere:

<img width="375" alt="image" src="https://user-images.githubusercontent.com/901169/51158891-6b122880-183b-11e9-807f-fd1def13608d.png">